deploy/helm/sumologic/values.yaml

```diff
@@ -3473,14 +3473,26 @@ metadata:
               paths = [
                 "/prometheus.metrics",
                 "/prometheus.metrics.apiserver",
-                "/prometheus.metrics.applications.apache",
+                "/prometheus.metrics.applications.nginx-ingress",
+                "/prometheus.metrics.applications.nginx",
+                "/prometheus.metrics.applications.redis",
                 "/prometheus.metrics.applications.jmx",
                 "/prometheus.metrics.applications.kafka",
                 "/prometheus.metrics.applications.mysql",
                 "/prometheus.metrics.applications.nginx",
                 "/prometheus.metrics.applications.nginx-ingress",
                 "/prometheus.metrics.applications.postgresql",
-                "/prometheus.metrics.applications.redis",
+                "/prometheus.metrics.applications.apache",
+                "/prometheus.metrics.applications.sqlserver",
+                "/prometheus.metrics.applications.haproxy",
+                "/prometheus.metrics.applications.cassandra",
+                "/prometheus.metrics.applications.mongodb",
+                "/prometheus.metrics.applications.rabbitmq",
+                "/prometheus.metrics.applications.tomcat",
+                "/prometheus.metrics.applications.varnish",
+                "/prometheus.metrics.applications.memcached",
+                "/prometheus.metrics.applications.elasticsearch",
+                "/prometheus.metrics.applications.activemq",
```

Comment:
Perhaps we can sort it alphabetically?

Reply:
I sorted it in prometheus remote write config order

Reply:
I can sort it alphabetically. WDYT?

Reply:
sorted alphabetically

Reply:
> I sorted it in prometheus remote write config order

This can get old very quickly 😆 while alphabet doesn't change too often :)

BTW does it make any difference in what order this is defined in?

Reply:
easier to spot if all remoteWrites have been added to the endpoints, but:
`cat /sumologic/deploy/helm/sumologic/values.yaml | grep prometheus.metrics.applications | grep -oP '/prometheus.metrics.applications.*' | sort`
can be helpful